docs/ocicrypt.md

```diff
@@ -1,15 +1,30 @@
 # OCIcrypt
 
+nerdctl supports encryption and decryption using [OCIcrypt](https://github.com/containers/ocicrypt)
+(aka [imgcrypt](https://github.com/containerd/imgcrypt) for containerd).
 
-See https://github.com/containerd/imgcrypt to learn further information.
+## JWE mode
 
-## Encryption
+### Encryption
 
-See https://github.com/containerd/imgcrypt
+Use `openssl` to create a private key (`mykey.pem`) and the corresponding public key (`mypubkey.pem`):
+```bash
+openssl genrsa -out mykey.pem
+openssl rsa -in mykey.pem -pubout -out mypubkey.pem
+```
+
+Use `nerdctl image encrypt` to create an encrypted image:
+```bash
+nerdctl image encrypt --recipient=jwe:mypubkey.pem --platform=linux/amd64,linux/arm64 foo example.com/foo:encrypted
+nerdctl push example.com/foo:encrypted
+```
 
-## Decryption
+CAUTION :warning:: This command only encrypt image layers, but does NOT encrypt [container config information such as `Env` and `Cmd`](https://github.com/opencontainers/image-spec/blob/v1.0.1/config.md#example).
+To see non-encrypted information, run `nerdctl image inspect --mode=native --platform=PLATFORM example.com/foo:encrypted` .
 
-### Configuration
+### Decryption
+
+#### Configuration
```

Comment:
minor nit, nonblocking: not sure if a usecase may come up where the `/etc` fs may not be writable, in that case, perhaps it may be useful to point to this [document](https://github.com/containerd/containerd/blob/6883c845959ba36f72a3ca36a510d209b2248386/docs/cri/decryption.md)

Reply:
fixed